src/main/java/com/newrelic/jfr/stacktrace/StackTraceBlob.java

```diff
@@ -17,8 +17,13 @@ public static String encode(RecordedStackTrace trace) {
       var frameData = new HashMap<String, String>();
       var frame = frames.get(i);
       var method = frame.getMethod();
-      var methodDesc = method.getType().getName() + "." + method.getName() + method.getDescriptor();
-      frameData.put("desc", methodDesc);
+      if (method != null) {
+        var methodDesc =
+            method.getType().getName() + "." + method.getName() + method.getDescriptor();
+        frameData.put("desc", methodDesc);
+      } else {
+        frameData.put("desc", "[missing]");
+      }
```

Comment:
Prefer to factor out a method that returns the description for a method (defaulting to `[missing]`) and eliminating the inverted `if` logic and duplicate `put()`, but also not a huge deal.

Reply:
Fixed.